Controller/Export/Base.php

```diff
@@ -136,4 +136,12 @@ public function export(AbstractCollection $collection)
         }
         return $result;
     }
+
+    /**
+     * @return NostoHelperScope
+     */
+    public function getNostoHelperScope(): NostoHelperScope
```

Comment:
Should we use type hints from now on?

Reply:
We can, but only if the method doesn't return null